dbt/adapters/redshift/relation_configs/materialized_view.py

```diff
@@ -199,11 +208,13 @@ def parse_relation_results(cls, relation_results: RelationResults) -> Dict:
                 {"dist": RedshiftDistConfig.parse_relation_results(materialized_view)}
             )
 
-        # TODO: this only shows the first column in the sort key
-        if materialized_view.get("sortkey1"):
-            config_dict.update(
-                {"sort": RedshiftSortConfig.parse_relation_results(materialized_view)}
-            )
+        column_descriptor = relation_results.get("column_descriptor")
+        if column_descriptor:
```

Comment:
Both of these `if` statements could be updated to use the walrus operator.

Reply:
fixed